app/components/pdf-image-preview/pdf-image-preview.component.ts

```diff
@@ -0,0 +1,101 @@
+import angular from "angular";
+import { getDocument, GlobalWorkerOptions, PDFPageProxy } from "pdfjs-dist";
+// Set the pdfjsLib worker source manually - https://github.com/mozilla/pdf.js/issues/8305
+import pdfjsWorker from "pdfjs-dist/build/pdf.worker.entry";
+
+interface IPDFImagePreviewBindings {
+  pdf: string | URL;
+  scale?: number;
+  canvasWidth?: number;
+  canvasHeight?: number;
+}
+
+interface IPDFImagePreviewCtrl extends IPDFImagePreviewBindings {
```

Comment:
"i" is fine for IPDFImagePreviewCtrl here since I can't think of a useful name right now